### DISCUSSION TITLE
Different deflection between blade tip and last node in BeamDyn

### DISCUSSION THREAD

**Discussion by HCKimDaniel** [OpenFAST/openfast#1272] [Q&A] [answered]
Hello, 

I have a question about the difference in deflection between blade tip and last node in BeamDyn (between TipTDxr and N9TDxr)

I expected that the blade tip deflections (TipTDxr, TipTDyr, and TipTDzr) would be the same as the deflections on the last node. (N9TDxr, N9TDyr, and N9TDzr when point 9 is the last node point). But it is different. 

In outListParameters.xlsx
N9TDxr		Sectional translational deflection (relative to the undeflected position) at Node 9 expressed in r
TipTDxr		Tip translational deflection (relative to the undeflected position) expressed in r

In my understanding, if Node 9 is the last node, then two parameters are the same. 

Can you help me to find out the reason why they are different? 

I am using the IEA15MW input files and OpenFAST v3.2.1

I attached the figure that shows the difference and input files. 
[Deflection.zip](https://github.com/OpenFAST/openfast/files/9728548/Deflection.zip)
![difference_Deflection](https://user-images.githubusercontent.com/92457112/194419218-f6588006-d2a1-4d73-bbe3-c60106195cb0.jpg)

**OpenFAST Version**
OpenFAST-v3.2.1
Compile Info:

Compiler: Microsoft Visual Studio Professional 2019 Version 16.11.1
Architecture: 64 bit
Precision: single
OpenMP: No

OS: Window

**Comment by jjonkman** [comment, marked as answer]
Dear @HCKimDaniel,

I agree that if output node 9 was the blade tip, then the two outputs would agree, but I don't see in your model that output node 9 is at the tip.  You've set `OutNd`(9) = 50.  And it looks like you are using trapezoidal quadrature (`quadrature` = 2), a trapezoidal quadrature refinement of 2 (`refine` = 2), and 34 input stations (`station_total` = 34).  Thus, the tip node would be node 34+33 = 67, which is different than 50.  (Please note that the trapezoidal quadrature is based on the input stations, not the key points, which are used to define the blade reference axis.)

Best regards,

**Reply by HCKimDaniel** [reply]
Dear @jjonkman 

Thank you for your prompt reply!

Best regards,
Daniel Kim

**Comment by mohsamaz** [comment]
Hello everyone,
I'm using BeamDyn for blade modeling and ElastoDyn for the tower in my simulation.
I understand that BeamDyn doesn't directly output a tip-to-tower clearance parameter. Could you recommend the most effective approach or method to accurately calculate this clearance?

Thank you for your insights and support!

**Reply by jjonkman** [reply]
Dear @mohsamaz,

BeamDyn has no knowledge of the tower deflection, so, it doesn't support a tip-to-tower clearance output.   But you likely also have AeroDyn enabled along with BeamDyn, and you can use AeroDyn's tower clearance output.  That is, AeroDyn output `BαNβClrnc` gives the tower clearance for output node Nβ of blade Bα.

Best regards,
